docs/products/kafka/howto/configure-topic-tiered-storage.rst

```diff
@@ -0,0 +1,57 @@
+Configuring tiered storage for topics
+===========================================================================
+
+Aiven for Apache Kafka® offers flexibility in configuring tiered storage and setting retention policies. This guide will walk you through the process of configuring tiered storage for individual topics, configuring local retention policies, and configuring retention policies at the service level.
+
+.. important:: 
+    
+   Aiven for Apache Kafka® tiered storage is an early availability feature, which means it has some restrictions on the functionality and service level agreement. It is intended for non-production environments, but you can test it with production-like workloads to assess the performance. To enable this feature, navigate to the :doc:`Feature preview </docs/platform/howto/feature-preview>` page within your user profile.
+
+Prerequisite
+------------
+* Tiered storage enabled for the Aiven for Apache Kafka service.
+
+Configure tiered storage for topics
+------------------------------------
+
+1. Access `Aiven console <https://console.aiven.io/>`_, select your project, and choose your Aiven for Apache Kafka service.
+2. From the left sidebar, select **Topics**.
+3. Here, you have the option to either add a new topic with tiered storage configuration or modify an existing topic to use tiered storage.
+
+For a new topic
+~~~~~~~~~~~~~~~
+
+1. From the **Topics** page, select **Add topic**.
+2. Enable advanced configurations by setting the **Do you want to enable advanced configuration?** option to **Yes**.
+3. In the **Topic advanced configuration** drop-down, choose ``remote_storage_enable``. This action will reveal the **Remote storage enabled** drop-down. 
+4. Select **True** to activate tiered storage for the topic.
+
+.. note:: 
+    If you leave the value as **Default**, it implies that tiered storage is enabled for this topic.
```

Comment:
I thought the default was not to use it, interesting.

Reply:
Yeah i think so. Default in the topic configs means "not set", meaning it's as if it was never touched by user. (Maybe it should be "Not set" instead of "Default" 🤔) In this case, as you need to explicitly enable remote storage per topic, the default is that it's not enabled. Does that make sense?